cloudbuild.yaml

```diff
@@ -256,9 +257,16 @@ steps:
         python3 test_frontend.py "127.0.0.1:8081"
         echo "pass" > /workspace/rag_frontend_result.txt
 
-        # Upload locally stored netflix dataset to GCS bucket mounted as /data
-        gsutil cp ./netflix_titles.csv gs://gke-aieco-rag-$SHORT_SHA-$_BUILD_ID/netflix-shows/netflix_titles.csv
-        ray job submit --working-dir . --address=http://127.0.0.1:8265 -- python ray_job.py
```

Comment:
Delete the ray_job.py file that is no longer needed

Reply:
Done!